### DISCUSSION TITLE
problem with logging in

### DISCUSSION THREAD

**Discussion by fotifotis** [DurgNomis-drol/ha_toyota#212] [General]
Good morning,
I installed the integration in HACS and then I wanted to add Toyota Connected Services in the add-ons, where I then enter my email address and password for MyT, but I get a message about incorrect email address or password.
They are correct - checked in the application and on the Toyota website.

**Comment by CM000n** [comment]
Hi @fotifotis, have you made sure that you are already using the latest [`MyToyota`](https://play.google.com/store/apps/details?id=com.toyota.oneapp.eu) app before updating? It sounds like you are probably still using the old [`MyT`](https://play.google.com/store/apps/details?id=app.mytoyota.toyota.com.mytoyota) app.

See also: https://github.com/DurgNomis-drol/ha_toyota/releases/tag/v2.0.0

There has already been some feedback here from people who did not read the release notes properly and were still using the old MyT app before the update. The error pattern was the same for them.

Or are there any other indications of a possible error in your Home Assistant logs?

**Reply by fotifotis** [reply]
I downloaded the integration in hacs 30 minutes earlier, the update information is from yesterday

I'm sending a screenshot, "yesterday" means yesterday ;)

<img width="1412" alt="image" src="https://github.com/DurgNomis-drol/ha_toyota/assets/93143322/af2de697-dcf4-4a76-acca-6fcc967942b3">

**Reply by CM000n** [reply]
Sorry, but I don't understand what you're trying to tell me.
